trappy/base.py

```diff
@@ -179,6 +179,12 @@ def append_data(self, time, comm, pid, cpu, line, data):
         self.line_array.append(line)
         self.data_array.append(data)
 
+    def conv_to_int(self, value):
```

Comment:
What we are really doing is parsing a string to an integer. I would name it parse_int(str)